### PR TITLE
Add CloudInit Guest Customization example guide

### PR DESCRIPTION
**Pull request by Didainius**
This PR adds an example to Guest Customization docs in our Guides section. It demonstrates verified execution of a script using CloudInit mechanism in Ubuntu 22.04.

### CODE REVIEW

**Review by dataclouder — APPROVED** [review]

File: website/docs/guides/vm_guest_customization.html.markdown [hunk 1]
```diff
@@ -105,3 +105,63 @@ After applying the configuration one should be able to login using `core` user a
...
+supplying CloudInit user data.
+
+```sh
+#!/bin/sh
+echo "Command executed as 'ubuntu' user" >> /tmp/setup.log
+sudo echo "Command executed as 'root' user" >> /tmp/setup.log
+```
+
+VM definition might be different, but key here is `guest_properties`. It will supply needed data for
+CloudInit (including the shell script in file `script.sh`)
```
Comment (lvirbalas) [inline website/docs/guides/vm_guest_customization.html.markdown:141]: Additionally, maybe add a sentence about using base64encode?

**Review by lvirbalas — APPROVED** [review]
One suggestion and LGTM! (Read in doc preview tool).

**Review by adambarreiro — APPROVED** [review]
Nice!

**Review by vbauzysvmware — APPROVED** [review]

File: website/docs/guides/vm_guest_customization.html.markdown (hunk 1 above)
Comment (Didainius) [reply website/docs/guides/vm_guest_customization.html.markdown]: Additionally mentioned base64 encoded file is requirement by CloudInit

**Review by mikeletux — APPROVED** [review]

**Pull request merged**